package.json

```diff
@@ -3,7 +3,7 @@
 	"author": "Automattic Inc.",
 	"private": true,
 	"productName": "Studio",
-	"version": "1.0.7",
+	"version": "1.1.0-beta.1",
```

Comment:
Per our versioning guidelines, it should be 1.1.0-beta1. In other case, it won't work on Windows.